demo/dynamic-font-size/src/index.js

```diff
@@ -0,0 +1,88 @@
+// Paper
+
+const paperContainer = document.getElementById('paper-container');
+
+const graph = new joint.dia.Graph({}, { cellNamespace: joint.shapes });
+const paper = new joint.dia.Paper({
+    model: graph,
+    cellViewNamespace: joint.shapes,
+    width: '100%',
+    height: '100%',
+    gridSize: 20,
+    drawGrid: { name: 'mesh' },
+    async: true,
+    sorting: joint.dia.Paper.sorting.APPROX,
+    background: { color: '#F3F7F6' }
+});
+
+paperContainer.appendChild(paper.el);
+
+const ResizeTool = joint.elementTools.Control.extend({
+    getPosition: function(view) {
+        const model = view.model;
+        const { width, height } = model.size();
+        return { x: width, y: height };
+    },
+    setPosition: function(view, coordinates) {
+        const model = view.model;
+        model.set(
```

Comment:
I would simplify this.
`model.position(Math.max(coordinates.x, 1), Math.max(coordinates.y, 1))`
No need to send `ui` /  `tool` flags in the demo.